draft-demarco-oauth-status-attestations.md

```diff
@@ -60,7 +60,7 @@ without requiring to query any third-party entities.
 Status Assertions ensure the integrity and trustworthiness of digital credentials, whether in JSON Web Tokens (JWT) or CBOR Web Tokens (CWT) format, certifying their validity and non-revocation status. They function similarly to OCSP Stapling, allowing wallet instances to present time-stamped assertions from the Credential Issuer.
 The approach defined in this specification allows the verification of credentials against any revocation, without direct queries to the issuer, enhancing privacy, reducing latency, and enabling offline verification. Essential for offline scenarios, Status Assertions validate digital credentials' validity, balancing scalability, security, and privacy without internet connectivity.
 
-
+Figure 1 illustrates the process by which a Wallet Instance requests a Status Assertion from the Credential Issuer and subsequently receives it.
```

Comment:
```suggestion
The figure below illustrates the process by which a Wallet Instance requests a Status Assertion from the Credential Issuer and subsequently receives it.
```

we already have the text under the figure defining its number.

Reply:
I usually prefer to enumerate the figures and then to specifically point to them in the text. But i guess since the image is right bellow this works as well.